models/DataObject/ClassDefinition/Exception/WriteException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Model\DataObject\ClassDefinition\Exception;
+
+class WriteException extends \Exception
```

Comment:
Since `WriteException` is also used by field-collections and object bricks, I'd move it to `Pimcore\Model\DataObject\Exception` and maybe rename it to `DefinitionWriteException`

Reply:
I renamed and moved the exception